src/services/extensionAdapter/HardHatExtensionAdapter.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2022. Consensys Software Inc. All rights reserved.
+// Licensed under the MIT license.
+
+import {buildContracts} from '@/commands/HardhatCommands';
+import {NotificationOptions} from '@/Constants';
+import {showNotification} from '@/helpers/userInteraction';
+import {IExtensionAdapter} from '@/services/extensionAdapter/IExtensionAdapter';
+import {Constants} from '@/constants';
```

Comment:
Here we should use the same casing, _i.e._, `@/Constants`. Otherwise, webpack gives a warning

```
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
```